guides/Step By Step Guide/CreatingRoleCommand.md

```diff
@@ -0,0 +1,1090 @@
+Excellent! Now that you have managed to understand how to transfer and edit the existing commands that Klasa made for us, it is time to make our very own command.
+
+Let's make a command that will allow server admins to give or take roles from a member.
+
+> Note: As always, you can copy and paste this final version snippet or you can follow along the guide step by step to see how we make the command.
+
+```ts
+const { Command } = require(`klasa`);
+const { Permissions } = require(`discord.js`);
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			aliases: [`r`, `ro`, `rol`],
+			bucket: 5,
+			cooldown: 60,
+			description: `Adds or removes a role from a member.`,
+			// eslint-disable-next-line
+			extendedHelp: `This command can add or remove a role from a member but it will require the user to have **MANAGE_GUILD** permissions to use. The command can be used in the following ways: **+role add @role**, **+role remove @role**, or **+role @role** if you want the bot to automatically add or remove based on if the member has the role already.`,
+			permissionLevel: 6,
+			requiredPermissions: [Permissions.FLAGS.MANAGE_ROLES],
```

Comment:
Using the string form is also allowed and has no performance drawbacks compared to this one, as this is stored as the parsed `Permissions` instance.
```suggestion
			requiredPermissions: ['MANAGE_ROLES'],
```